shader/draw_leaf.wgsl

```diff
@@ -242,7 +244,7 @@ fn main(
         if tag_word == DRAWTAG_BEGIN_CLIP {
             path_ix = m.path_ix;
         }
-        clip_inp[m.clip_ix] = ClipInp(ix, i32(path_ix));
+        clip_inp[min(m.clip_ix, config.n_clip - 1u)] = ClipInp(ix, i32(path_ix));
```

Comment:
This will be gated by the tag being NOP. Further, it's the locus of a potentially more severe set of problems. In the likely case that you get a whole bunch of END_CLIP tags, they'll all pile on to the last element in the `clip_inp` array, which will leave it with a likely incorrect value, and also be undefined behavior due to racy writes.